README.md

```diff
@@ -281,6 +281,40 @@ class Shop < ActiveRecord::Base
 end
 ```
 
+### Access Scopes Strategies
+The Shopify App gem now provides handling changes to scopes for both shop/offline and user/online tokens. By default, the `ShopifyApp::AccessScopes::ShopStrategy` and `ShopifyApp::AccessScopes::UserStrategy` strategies are used on app loads and during OAuth.
```

Comment:
Write 'now' here implies a recent change – which is true today, but will become less true over time, so I think should be dropped.